libs/atqr/domain/src/lib/domain-spec/aggregrates/Challenge Aggregate.disl

```diff
@@ -30,40 +31,41 @@ domain ATQR
           @deadline: Date
           @pledge: DISL.Money
             greater than 0
-          @deadline: FutureDate
           @requiredProof?: DISL.Paragraph
           /**
            * Allows contenders to enroll using a link instead of being invited
            * @param {yes or no} yes if contenders can enroll using a link
            */
           @inviteByLink: yes or no
-          @enrollmentDeadline: FutureDate
-            before @deadline
-          @owner: Contact
-
-          //TODO: this "property" below was added to User entity as a property. Can it now be deleted from here?
-          ownerPaymentMethod: PaymentMethod | null
+          @paymentMethod: PaymentMethod | null
 
           when @createdByContender
-
-            ⚙️ registerContender :name :email
-              rules:
-                when Challenge @status is not .Unverified do nothing
-
-              find_or_create new User
+          ⚙️ registerPlayer :name :email
+            rules:
+              find_or_create User with ^email
                 it's paymentMethod = PaymentMethod
                 createEnrollmentFor User
 
-                  on SUCCESS:
-                    show ui
-                      CheckEmailToConfirmParticipation
+                on SUCCESS:
+                  return "Player registered"
 
-                  on FAILURE:
-                    throw "Could not create contender"
+                on FAILURE:
+                  throw "Could not create player"
 
-        ⚙️ verifyIfChallengeCanBegin :contenders :judges :paymentMethods
+          when @createdByJudge
+          ⚙️ registerPlayer :name :email
```

Comment:
This function should not be inside recipe and Player does not exists in our api anymore